dds_registration/models.py

```diff
@@ -109,7 +124,8 @@ class Event(models.Model):
     description = models.TextField(blank=True)
     public = models.BooleanField(default=False)
     registration_open = models.DateField(auto_now_add=True, help_text='Date registration opens (inclusive)')
-    registration_close = models.DateField(blank=True, help_text='Date registration closes (inclusive)')
+    # Returned `null=True` here to allow to store an 'empty' dates. Is that ok?
```

Comment:
No, every event needs to have a close of registration - at the latest when the event finishes.

Reply:
> No, every event needs to have a close of registration - at the latest when the event finishes.

But in this case we should remove `blank=True` too, right?

Reply:
Yes, thanks.